### PR TITLE
Issue/125

### LINKED ISSUE #125: Document API endpoints
All the endpoints available in the API and what they do. This could possibly just be a link to the swagger page that already documents it. It can be accessed at http://localhost:5000/swagger/index.html when hosting the API locally.

It can also be accessed on [production](http://srv.giraf.cs.aau.dk/DEV/API/swagger/index.html), however the relevant JSON file is missing, which produces an error.

### PR DESCRIPTION
**Pull request by Huften**
A file for each end point category was made. These are located in \wiki\docs\development\rest_api_development\endpoints_documentation. Added header, URL endpoint, a short description, request and/or response body and response status codes. This was added to every endpoint from swagger.

closes #125

### CODE REVIEW

**Comment by Determinatoro** [comment]
Please use the following checklist, @undefined. Feel free to check it off as you go along.

**Documentation**

- [ ] The documentation is well structured.
- [ ] The documentation is understandable.
- [ ] The documentation is not missing information.

**Rendering the WiKi**

- [ ] There are no dead links or references.
- [ ] The markdown renders correctly in MkDocs.
- [ ] The content is placed in the correct page/folder/section.

**Comment by Determinatoro** [comment]
Please use the following checklist, @undefined. Feel free to check it off as you go along.

**Documentation**

- [ ] The documentation is well structured.
- [ ] The documentation is understandable.
- [ ] The documentation is not missing information.

**Rendering the WiKi**

- [ ] There are no dead links or references.
- [ ] The markdown renders correctly in MkDocs.
- [ ] The content is placed in the correct page/folder/section.

**Comment by Determinatoro** [comment]
Please use the following checklist, @undefined. Feel free to check it off as you go along.

**Documentation**

- [ ] The documentation is well structured.
- [ ] The documentation is understandable.
- [ ] The documentation is not missing information.

**Rendering the WiKi**

- [ ] There are no dead links or references.
- [ ] The markdown renders correctly in MkDocs.
- [ ] The content is placed in the correct page/folder/section.

**Comment by Determinatoro** [comment]
Please use the following checklist, @Riasalit. Feel free to check it off as you go along.

**Documentation**

- [x] The documentation is well structured.
- [x] The documentation is understandable.
- [x] The documentation is not missing information.

**Rendering the WiKi**

- [x] There are no dead links or references.
- [x] The markdown renders correctly in MkDocs.
- [x] The content is placed in the correct page/folder/section.

**Comment by Determinatoro** [comment]
Please use the following checklist, @Emil7370. Feel free to check it off as you go along.

**Documentation**

- [x] The documentation is well structured.
- [x] The documentation is understandable.
- [ ] The documentation is not missing information.

**Rendering the WiKi**

- [x] There are no dead links or references.
- [x] The markdown renders correctly in MkDocs.
- [x] The content is placed in the correct page/folder/section.

File: docs/development/rest_api_development/endpoints_documentation/AccountEndpoints.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Login
+
+```
+
+ /v1/Account/login
+
+```
+
+This endpoint allows the user to sign in to his/her account by providing valid username and password.\
```
Comment (Riasalit) [inline docs/development/rest_api_development/endpoints_documentation/AccountEndpoints.md]: random "\\"

**Review by Riasalit — CHANGES_REQUESTED** [review]
1 wrong backslash, otherwise its good

**Review by Riasalit — APPROVED** [review]

**Review by Emil7370 — APPROVED** [review]

**Pull request merged**